### DISCUSSION TITLE
How can i host my presentation on vercel

### DISCUSSION THREAD

**Discussion by veritem** [slidevjs/slidev#249] [Ideas]
The build command doesn't work for me, it always fails

**Comment by antfu** [comment]
What's the error and what the package manager you are using?

**Reply by veritem** [reply]
My package manager is `pnpm`  

When i built the build failed because it required this [package](https://www.npmjs.com/package/playwright-chromium) and its installation takes forever.

**Reply by antfu** [reply]
You do you have this?

```yaml
---
download: true
---
```

If not, set `PLAYWRIGHT_SKIP_BROWSER_DOWNLOAD=1` env to skip the downloading (as you don't need it)

**Reply by veritem** [reply]
"If not, set PLAYWRIGHT_SKIP_BROWSER_DOWNLOAD=1 env to skip the downloading (as you don't need it)"  This didn't work

**Reply by veritem** [reply]
i added an `.env` file and added this like

```bash
PLAYWRIGHT_SKIP_BROWSER_DOWNLOAD=1
```

**Comment by cnguyen-de** [comment]
A bit late, but here are the configs for vercel:

![Screenshot 2022-10-13 at 12 20 32](https://user-images.githubusercontent.com/6671583/195571612-2388fec5-b1e7-47d2-987c-f00a3c183e87.png)

Configure your build command to target the correct .md file that you're using. Playwright-chromium should also be in devDependencies.
